test/libsolidity/ASTJSON/ast_internal_function_different_ids_export.json

```diff
@@ -0,0 +1,805 @@
+        "src": "34:2:1"
+      },
+      "returnParameters":
+      {
+        "id": 6,
+        "nodeType": "ParameterList",
+        "parameters": [],
+        "src": "37:0:1"
+      },
+      "scope": 64,
+      "src": "20:19:1",
+      "stateMutability": "nonpayable",
+      "virtual": false,
+      "visibility": "internal"
+    },
+    {
+      "abstract": false,
+      "baseContracts": [],
+      "canonicalName": "L",
+      "contractDependencies": [],
+      "contractKind": "library",
+      "fullyImplemented": true,
+      "id": 17,
+      "linearizedBaseContracts":
+      [
+        17
+      ],
+      "name": "L",
+      "nameLocation": "48:1:1",
+      "nodeType": "ContractDefinition",
+      "nodes":
+      [
+        {
+          "body":
+          {
+            "id": 11,
+            "nodeType": "Block",
+            "src": "78:2:1",
+            "statements": []
+          },
+          "id": 12,
+          "implemented": true,
+          "kind": "function",
+          "modifiers": [],
+          "name": "g",
+          "nameLocation": "65:1:1",
+          "nodeType": "FunctionDefinition",
+          "parameters":
+          {
+            "id": 9,
+            "nodeType": "ParameterList",
+            "parameters": [],
+            "src": "66:2:1"
+          },
+          "returnParameters":
+          {
+            "id": 10,
+            "nodeType": "ParameterList",
+            "parameters": [],
+            "src": "78:0:1"
+          },
+          "scope": 17,
+          "src": "56:24:1",
+          "stateMutability": "nonpayable",
+          "virtual": false,
+          "visibility": "internal"
+        },
+        {
+          "body":
+          {
+            "id": 15,
+            "nodeType": "Block",
+            "src": "107:2:1",
+            "statements": []
+          },
+          "id": 16,
+          "implemented": true,
+          "kind": "function",
+          "modifiers": [],
+          "name": "h",
+          "nameLocation": "94:1:1",
+          "nodeType": "FunctionDefinition",
+          "parameters":
+          {
+            "id": 13,
+            "nodeType": "ParameterList",
+            "parameters": [],
+            "src": "95:2:1"
+          },
+          "returnParameters":
+          {
+            "id": 14,
+            "nodeType": "ParameterList",
+            "parameters": [],
+            "src": "107:0:1"
+          },
+          "scope": 17,
+          "src": "85:24:1",
+          "stateMutability": "nonpayable",
+          "virtual": false,
+          "visibility": "internal"
+        }
+      ],
+      "scope": 64,
+      "src": "40:71:1",
+      "usedErrors": []
+    },
+    {
+      "abstract": false,
+      "baseContracts": [],
+      "canonicalName": "C",
+      "contractDependencies": [],
+      "contractKind": "contract",
+      "fullyImplemented": true,
+      "id": 38,
+      "internalFunctionIDs":
+      {
+        "12": 2,
+        "16": 3,
+        "8": 1
+      },
```

Comment:
I added this other test case to explicitly check the different internal function IDs on two different contracts.